test/integration/scheduler/priorities_test.go

```diff
@@ -39,7 +39,6 @@ import (
 )
 
 // This file tests the scheduler priority functions.
-
```

Comment:
restore this line so that the comment is not associated to the function below.

Reply:
oops... Updated.